.vscode/settings.json

```diff
@@ -69,4 +69,5 @@
     "black-formatter.args": [
         "-l 120"
     ],
+    "search.useIgnoreFiles": false,
```

Comment:
Can we justify enabling this for all users? I feel like build files should not show up in search by default as they're only relevant if you're debugging the build system